JavascriptFormatter.js

```diff
@@ -548,8 +543,18 @@ function formatCommand(command) {
                             eq = eq.invert();
                         line = waitFor(eq);
                     }
+                    else if (command.command.match(/^(getEval|runScript)/)) {
+                        call = new CallSelenium(def.name, xlateArgument(command.getParameterAt(0)), command.getParameterAt(0));
```

Comment:
the second param is equal to the first?
command.getParameterAt(0),command.getParameterAt(0)? A typo?